src/content/developers/docs/oracles/index.md

```diff
@@ -14,7 +14,7 @@ Make sure you're familiar with [nodes](/developers/docs/nodes-and-clients/), [co
 
 ## What is an oracle {#what-is-an-oracle}
 
-An oracle is a bridge between the blockchain and the real world. They act as on-chain APIs you can query to get information into your smart contracts. This could be anything from price information to weather reports.
+An oracle is a bridge between the blockchain and the real world. They act as on-chain APIs you can query to get information into your smart contracts. This could be anything from price information to weather reports. Oracles are also bi-directional, and they can also be used to "send" data out to the real world. 
```

Comment:
```suggestion
An oracle is a bridge between the blockchain and the real world. They act as on-chain APIs you can query to get information into your smart contracts. This could be anything from price information to weather reports. Oracles can also be bi-directional, used to "send" data out to the real world. 
```